spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/autoconfiguration/XsuaaAutoConfigurationTest.java

```diff
@@ -57,6 +57,7 @@ public void configures_xsuaaRestTemplate() {
 	public void configures_xsuaaServiceConfiguration_withProperties() {
 		contextRunner
 				.withPropertyValues("spring.xsuaa.auto:true")
+				.withPropertyValues("spring.xsuaa.disable-default-property-source:false")
```

Comment:
please remove this to check that by default property-source is configured :-)

Reply:
That is why it is set to false. False means it is **not disabeld**.